django/chat/responses.py

```diff
@@ -169,26 +170,43 @@ def summarize_response(chat, response_message):
     model = chat.options.summarize_model
 
     llm = OttoLLM(model)
+    error_str = ""
 
     if len(files) > 0:
         titles = [file.filename for file in files]
         responses = []
         for file in files:
+            if cache.get(f"stop_response_{response_message.id}", False):
+                break
```

Comment:
Moving this up here will break faster; reversing the condition so we can lose a level of indentation